account_statement_import_base/models/account_bank_statement_line.py

```diff
@@ -10,8 +10,8 @@ class AccountBankStatementLine(models.Model):
 
     # Ensure transactions can be imported only once
     # if the import format provides unique transaction IDs
-    unique_import_id = fields.Char(string="Import ID", readonly=True, copy=False)
-    raw_data = fields.Text(readonly=True, copy=False)
+    unique_import_id = fields.Char(string="Import ID", copy=False)
+    raw_data = fields.Text(copy=False)
```

Comment:
Any reason to move `readonly` from here to the view?

Reply:
yes, there is a quote in the document, see in: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0

> If a field is marked as readonly=True on the model, it won't be possible to import it through the Odoo import tool, so avoid it as possible and define the readonly attribute on the views instead.